Automattic/jetpack
# AI Assistant: Add Gutenberg syntax parsing

Extracted from #31708

## Proposed changes:

* Adds the Gutenberg parsing

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block
* Enable Gutenberg syntax
* Send a request
* Check that the blocks are parsed and inserted as children blocks
